graphql/host_resolver.go

```diff
@@ -15,6 +15,23 @@ import (
 	"github.com/mongodb/grip/message"
 )
 
+// Ami is the resolver for the ami field.
+func (r *hostResolver) Ami(ctx context.Context, obj *restModel.APIHost) (*string, error) {
+	host, err := host.FindOneId(utility.FromStringPtr(obj.Id))
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error finding host %s: %s", utility.FromStringPtr(obj.Id), err.Error()))
+	}
+	if host == nil {
+		// This should never happen, but if it does, we should log it.
+		grip.Warning(message.Fields{
```

Comment:
I'm not sure if this warrants logging at all or if we should even log at a higher level. If this ever happens something is probably wrong with the host.

Reply:
we log graphql errors anyway right? so idk about an additional log.